feature-sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Sessions.kt

```diff
@@ -302,6 +302,12 @@ fun SessionsTopBar(
         (scheduleState as? Loaded)?.schedule?.days?.let { days ->
             TabRow(
                 selectedTabIndex = pagerState.currentPage,
+                modifier = Modifier
+                    .background(
+                        color = MaterialTheme.colorScheme
+                            .surfaceColorAtElevation(2.dp)
+                    )
```

Comment:
[memo]
If the background is not set, a plain background is displayed.

<img src="https://user-images.githubusercontent.com/50792596/188673032-119dce90-2c2e-4260-a916-efc60595e221.png" width="300" />

Reply:
Thank you for your dedicated 😉